identity/app/views/profile/emailPrefs.scala.html

```diff
@@ -12,7 +12,9 @@
                 "email-subscription--subscribed" -> subscription.subscribedTo,
                 "email-subscription--first" -> row.isFirst))">
                 @if(subscription.subscribedTo){<input type="hidden" name="emailSubscription[]" value="@subscription.listId" />}
-                <div class="email-subscription__name">@subscription.name</div>
+                <div class="email-subscription__name">@subscription.name
+                    @if(subscription.subheading.isDefined){<br>@subscription.subheading}
```

Comment:
I would also suggest not using a `<br>` tag if its just for formatting reasons. You could change the markup so `subscription.subheading` has its own tag (maybe use a `h2`, and `h3`)